### ISSUE TITLE
No icon on desktop or task bar

### ISSUE THREAD

**Issue by Danny3** [Umio-Yasuno/amdgpu_top#37]
Hello!

For the programs that i have installed I prefer to have icons too, besides their names and descriptions, so that they are more user friendly, easier / faster to recognize and I can pin them to the start menu's Favorites section, the desktop or the task bar for quick access.

Unfortunately for this program, the GUI doesn't have any icon when displayed on both the start menu and on the desktop and it has one on the task bar when it's opened, which looks like a white "W" on a yellow filled circle  shape background, which I assume it's some kind of Wayland default when there's no icon or something is wrong as I've seen this icon a few times before for other programs

I tested this program (v0.1.10 ) on Debian 12 + KDE Plasma 5.27.5 (on Wayland session)

Can you please add an icon for it?

For me even a simple one like just with the letters "AT" or "AGT" (from AmdgpuTop or AmdGpuTop) would suffice.
And if you make it, could you make it in the SVG format so that it scales properly to any resolution?

Thank you!

**Comment by Umio-Yasuno** [comment]
I will not add an icon to `amdgpu_top`.
I think `amdgpu_top` is a small tool, So I don't want to add icons that make people aware of brands and such.

If you want the icon on the desktop entry for `amdgpu`, edit your local desktop entry and add `Icon=<other_application_name>`.

**Comment by Danny3** [comment]
> I will not add an icon to `amdgpu_top`. I think `amdgpu_top` is a small tool, So I don't want to add icons that make people aware of brands and such.
> 
> If you want the icon on the desktop entry for `amdgpu_top`, edit your local desktop entry and add `Icon=<other_application_name>`.

I understand!

For this kind of cases, after creating the shortcut on the Desktop with "Add to Desktop" in KDE Plasma...
I right-click the created shortcut and choose "Properties".
After that I just click on the icon placeholder, that says "Select Icon..." and then choose one from that huge list of icons.
The icon placeholder changes to the icon I've just chose so I press "OK" and the window closes.
If I open the desktop entry with Kate text editor I see that "Icon=" line pointing to the icon I've chosen.

For this case I tried to do the same since there's no icon anyway, but at the last step, after the placeholder is updated with the icon I've chosen, when I press OK to save the changes and close that properties window, the following Error pop-up appears:

"Could not save properties due to insufficient write access to:
'/home/<my-username>/Desktop/amdgpu_top.desktop

I have to press "OK" on this error pop-up window to close it and "Cancel" on the other one as pressing "OK" will always get into this error.

Any idea why this happens, what permissions it needs and why?

Opening the .desktop file with Kate text editor and adding that line will probably work as even if it requires permission, Kate normally asks for authentication, but I don't know what what is that icon that I'm trying to set called and where is it stored.

I don't know where that the pop-up windows takes all the icons when I tried the first way with clicking on that placeholder icon.

I haven't tried yet downloading the .desktop files and put them in that folder that you said in the other issue.

But I wanted to let you know about this problem anyway as some people might try like me to fix it with the tools that KDE Plasma already provides for adding / changing icons for programs.
